packages/features/bookings/lib/handleNewBooking/loadUsers.ts

```diff
@@ -0,0 +1,102 @@
+import { Prisma } from "@prisma/client";
+import type { IncomingMessage } from "http";
+
+import { orgDomainConfig } from "@calcom/features/ee/organizations/lib/orgDomains";
+import { HttpError } from "@calcom/lib/http-error";
+import logger from "@calcom/lib/logger";
+import { UserRepository } from "@calcom/lib/server/repository/user";
+import prisma, { userSelect } from "@calcom/prisma";
+import { credentialForCalendarServiceSelect } from "@calcom/prisma/selects/credential";
+
+import type { NewBookingEventType } from "../handleNewBooking";
+
+const log = logger.getSubLogger({ prefix: ["[loadUsers]:handleNewBooking "] });
+type Hosts = NewBookingEventType["hosts"];
+
+export const loadUsers = async (
+  eventType: NewBookingEventType,
```

Comment:
nit: I think we could simplify this type to pick or have just `hosts` and `users`